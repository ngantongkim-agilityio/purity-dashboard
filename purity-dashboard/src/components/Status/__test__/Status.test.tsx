import { render } from '@testing-library/react';

// Components
import { Status } from '..';

describe('Status component', () => {
  it('Should render Status component with default props', () => {
    const { container } = render(<Status />);

    expect(container).toMatchSnapshot();
  });
});
