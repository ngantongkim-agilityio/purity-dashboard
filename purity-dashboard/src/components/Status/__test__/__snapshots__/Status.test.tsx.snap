// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status component Should render Status component with default props 1`] = `
<div>
  <span
    class="inline-flex items-center rounded-full px-2 py-1 text-xs bg-gray-100 text-gray-500"
  >
    Pending
    <svg
      aria-hidden="true"
      class="ml-1 w-4 text-gray-500"
      data-slot="icon"
      fill="none"
      stroke="currentColor"
      stroke-width="1.5"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12 6v6h4.5m4.5 0a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
  </span>
</div>
`;
